api_tests/lib/actor.ts

```diff
@@ -182,24 +185,28 @@ export class Actor {
                     let diff = min_median_block_time - currentMedianBlockTime;
 
                     if (diff > 0) {
-                        console.log(
-                            `Waiting for median time to pass %d`,
-                            min_median_block_time
-                        );
+                        if (global.verbose) {
+                            console.log(
+                                `Waiting for median time to pass %d`,
+                                min_median_block_time
+                            );
+                        }
 
                         let escapeSequence = "";
 
                         while (diff > 0) {
-                            await sleep(1000);
```

Comment:
📣Our HTLC timeouts are set to +3/+8 seconds, so asking the chain every 200ms for the latest time won't make much difference.